demo/demo.html

```diff
@@ -29,12 +30,87 @@
 		</script>
 	</head>
 	<body>
-		<div class="demo1" data-init="(function($elm) { $elm.plugin(); })" data-resources="[{test: (function() {return true;}), paths: ['js/jquery.plugin.js', 'css/demo.css', 'css/demo1.css']}]">
+<!--
```

Comment:
indentation seems to be broken. mix of tabs and spaces. can you place our default .editorconfig in this repo and reformat?